infra/main.tf

```diff
@@ -0,0 +1,107 @@
+resource "azurerm_resource_group" "nachet-resources-group" {
+  name     = "nachet"
```

Comment:
naming doesn't match our current ADR. either we change the ADR or we make this match the ADR:

https://github.com/ai-cfia/dev-rel-docs/pull/77/files